doc/nrf/includes/sample_board_rows.txt

```diff
@@ -230,6 +230,10 @@
 
 | :ref:`nRF54L15 DK <ug_nrf54l>` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l15/cpuapp`` |
 
+.. nrf54l15dk_nrf54l10_cpuapp_ns
+
+| :ref:`nRF54L15 DK <ug_nrf54l>` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l10>` | ``nrf54l15dk/nrf54l10/cpuapp/ns`` |
+
```

Comment:
The cpu_app and cpu_app entries could be combined as like the entry in line 237.
```suggestion
.. nrf54l15dk_nrf54l10_cpuapp_and_cpuapp_ns

| :ref:`nRF54L15 DK (emulating nRF54L10) <ug_nrf54l>` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l10>` | ``nrf54l15dk/nrf54l10/cpuapp``    |
|                                                     |          |                                                |                                   |
|                                                     |          |                                                | ``nrf54l15dk/nrf54l10/cpuapp/ns`` |

```